Dockerfile

```diff
@@ -102,4 +102,4 @@ ENV HYRAX_ENGINE_PATH /app/samvera/hyrax-engine
 COPY --chown=1001:101 $APP_PATH /app/samvera/hyrax-webapp
 COPY --chown=1001:101 . /app/samvera/hyrax-engine
 
-RUN bundle install --jobs "$(nproc)"
+RUN bundle install --jobs "$(nproc)"
```

Comment:
```suggestion
RUN bundle install --jobs "$(nproc)"

```

Reply:
this suggestion might be a little wonky. what i'm getting at is that there aren't any real changes left in this file, so we should be able to drop it from the diff.